### ISSUE TITLE
[RFE] Mouselook should be enabled by default

### ISSUE THREAD

**Issue by AsciiWolf** [flathub/com.realm667.WolfenDoom_Blade_of_Agony#3]
I think that the "Always Mouselook" option should be enabled by default. And maybe some other tweaks (WASD keys?).

**Comment by Alexander-Wilms** [comment]
Yeah, that makes sense. But how could one implement this without overriding the player's settings? OTOH I guess nobody will use anything but WASD + mouselook. So I could write a script that sets these options in gzdoom.ini each time the game is started

**Comment by AsciiWolf** [comment]
@Alexander-Wilms Maybe you could ship the gzdoom.ini file for new installations [like I did](https://github.com/flathub/com.moddb.TotalChaos/blob/master/com.moddb.TotalChaos.yaml#L68) with my Total Chaos Flatpak? :-)

**Comment by AsciiWolf** [comment]
@Alexander-Wilms In this case, it is the gzdoom.ini shipped in the official Total Chaos zip file. But yep, it's basically a default ini with some changes.
